spec/jobs/education_form/create_daily_spool_files_spec.rb

```diff
@@ -55,7 +55,7 @@
       mock_file = double(File)
       mock_writer = StringIO.new
       sftp_mock = double(file: mock_file)
-      Net::SFTP.stub(:start).and_yield(sftp_mock)
+      expect(Net::SFTP).to receive(:start).once.and_yield(sftp_mock)
```

Comment:
Why? it's not a method or class you have created? There shouldn't be any reason to test expectation for code that you didn't write.